tools/ctl/ide/commands/commands.go

```diff
@@ -51,21 +52,66 @@ const (
 	RundexReadParallelism = 10
 	LLMRequestParallelism = 50
 	expertPrompt          = `You are an expert in diagnosing build issues in multiple open source ecosystems. You will help diagnose why builds failed, or why the builds might have produced an artifact that differs from the upstream open source package. Provide clear and concise explantions of why the rebuild failed, and suggest changes that could fix the rebuild`
+	containerName         = "oss-rebuild-debug-temp"
 )
 
-func NewRebuildCmds(app *tview.Application, rb *rebuilder.Rebuilder, modalFn modal.Fn, butler localfiles.Butler, aiClient *genai.Client, buildDefs rebuild.LocatableAssetStore, dex rundex.Reader, benches benchmark.Repository, cmdReg *commandreg.Registry) []commandreg.RebuildCmd {
+func runLocalInput(ctx context.Context, executor *local.DockerRunExecutor, dex rundex.Reader, inp rebuild.Input) error {
```

Comment:
can we just make this runLocal and have the other be inferLocal?

if you think it necessary, you could then write inferAndRunLocal which composes the two.